core/controllers/profile.py

```diff
@@ -76,16 +78,37 @@ def get(self, username):
         user_settings = user_services.get_user_settings_from_username(username)
         if not user_settings:
             raise self.PageNotFoundException
+        
+        user_contributions = user_services.get_user_contributions(
+            user_settings.user_id)
 
         self.values.update({
             'user_bio': user_settings.user_bio,
             'first_contribution_datetime': (
-                utils.get_time_in_millisecs(user_settings.first_contribution_datetime)
+                utils.get_time_in_millisecs(
+                    user_settings.first_contribution_datetime)
                 if user_settings.first_contribution_datetime else None),
             'profile_picture_data_url': user_settings.profile_picture_data_url,
+            'created_explorations_count': len(
+                user_contributions.created_explorations),
+            'edited_explorations_count': len(
+                user_contributions.edited_explorations)
+
         })
         self.render_json(self.values)
 
+    def get_public_contributions(list_explorations):
```

Comment:
Ah -- Travis should not have to write any backend code. It should be possible for him to just call the URLs you expose and get all the data he needs by doing that.